docs/index.md

```diff
@@ -1,6 +1,8 @@
 ---
 hide:
-  - navigation
+
+    - navigation
+
```

Comment:
revert?

Reply:
This is one of the markdownlint rules that was enabled: `plugins.md032.enabled = true # blanks-around-lists`